windows/tools/tag_scanner_window.py

```diff
@@ -369,6 +358,7 @@ def scan(self):
 
                 except Exception:
                     print("    Could not scan '%s'" % tag.filepath)
+                    print(format_exc())

```

Comment:
A print of the stack trace should always be before any "summary" statement. The user will see a big stack trace, and the thing they'll usually do is look at the bottom for the tl; dr